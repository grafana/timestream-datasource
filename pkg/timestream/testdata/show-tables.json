{
    "ColumnInfo": [
        {
            "Name": "Table",
            "Type": {
                "ArrayColumnInfo": null,
                "RowColumnInfo": null,
                "ScalarType": "VARCHAR",
                "TimeSeriesMeasureValueColumnInfo": null
            }
        }
    ],
    "NextToken": null,
    "QueryId": "#QueryId#",
    "QueryStatus": {
        "CumulativeBytesMetered": 0,
        "CumulativeBytesScanned": 0,
        "ProgressPercentage": 100
    },
    "Rows": [
        {
            "Data": [
                {
                    "ArrayValue": null,
                    "NullValue": null,
                    "RowValue": null,
                    "ScalarValue": "DevOps",
                    "TimeSeriesValue": null
                }
            ]
        },
        {
            "Data": [
                {
                    "ArrayValue": null,
                    "NullValue": null,
                    "RowValue": null,
                    "ScalarValue": "IoT",
                    "TimeSeriesValue": null
                }
            ]
        }
    ]
}