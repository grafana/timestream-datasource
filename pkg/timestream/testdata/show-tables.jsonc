//  ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ
//  
//  Frame[0] {
//      "typeVersion": [
//          0,
//          0
//      ]
//  }
//  Name: 
//  Dimensions: 1 Fields by 2 Rows
//  +-----------------+
//  | Name: Table     |
//  | Labels:         |
//  | Type: []*string |
//  +-----------------+
//  | DevOps          |
//  | IoT             |
//  +-----------------+
//  
//  
//  ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ
{
  "status": 200,
  "frames": [
    {
      "schema": {
        "meta": {
          "typeVersion": [
            0,
            0
          ]
        },
        "fields": [
          {
            "name": "Table",
            "type": "string",
            "typeInfo": {
              "frame": "string",
              "nullable": true
            }
          }
        ]
      },
      "data": {
        "values": [
          [
            "DevOps",
            "IoT"
          ]
        ]
      }
    }
  ]
}