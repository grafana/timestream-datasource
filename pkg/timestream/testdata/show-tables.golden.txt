ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ

Frame[0] {}
Name: 
Dimensions: 1 Fields by 2 Rows
+-----------------+
| Name: Table     |
| Labels:         |
| Type: []*string |
+-----------------+
| DevOps          |
| IoT             |
+-----------------+


====== TEST DATA RESPONSE (arrow base64) ======
FRAME=QVJST1cxAAD/////KAEAABAAAAAAAAoADgAMAAsABAAKAAAAFAAAAAAAAAEDAAoADAAAAAgABAAKAAAACAAAAHQAAAADAAAATAAAACgAAAAEAAAAZP///wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAACE////CAAAAAwAAAAAAAAAAAAAAAQAAABuYW1lAAAAAKT///8IAAAADAAAAAIAAAB7fQAABAAAAG1ldGEAAAAAAQAAABgAAAAAABIAGAAUABMAEgAMAAAACAAEABIAAAAUAAAARAAAAEgAAAAAAAUBRAAAAAEAAAAMAAAACAAMAAgABAAIAAAACAAAABAAAAAFAAAAVGFibGUAAAAEAAAAbmFtZQAAAAAAAAAABAAEAAQAAAAFAAAAVGFibGUAAAAAAAAA/////5gAAAAUAAAAAAAAAAwAFgAUABMADAAEAAwAAAAgAAAAAAAAABQAAAAAAAADAwAKABgADAAIAAQACgAAABQAAABIAAAAAgAAAAAAAAAAAAAAAwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABAAAAAAAAAAEAAAAAAAAAAQAAAAAAAAAAAAAAABAAAAAgAAAAAAAAAAAAAAAAAAAAAAAAAGAAAACQAAAAAAAABEZXZPcHNJb1QAAAAAAAAAEAAAAAwAFAASAAwACAAEAAwAAAAQAAAALAAAADgAAAAAAAMAAQAAADgBAAAAAAAAoAAAAAAAAAAgAAAAAAAAAAAAAAAAAAAAAAAKAAwAAAAIAAQACgAAAAgAAAB0AAAAAwAAAEwAAAAoAAAABAAAAGT///8IAAAADAAAAAAAAAAAAAAABQAAAHJlZklkAAAAhP///wgAAAAMAAAAAAAAAAAAAAAEAAAAbmFtZQAAAACk////CAAAAAwAAAACAAAAe30AAAQAAABtZXRhAAAAAAEAAAAYAAAAAAASABgAFAATABIADAAAAAgABAASAAAAFAAAAEQAAABIAAAAAAAFAUQAAAABAAAADAAAAAgADAAIAAQACAAAAAgAAAAQAAAABQAAAFRhYmxlAAAABAAAAG5hbWUAAAAAAAAAAAQABAAEAAAABQAAAFRhYmxlAAAAUAEAAEFSUk9XMQ==
